scale-encode-derive/src/lib.rs

```diff
@@ -255,3 +265,16 @@ impl TopLevelAttrs {
         Ok(res)
     }
 }
+
+// Since we only care about `skip` at the moment, we just expose this helper,
```

Comment:
I had some troubles to understand 

```suggestion
// Checks if the attributes contain `skip`.
//
// NOTE: Since we only care about `skip` at the moment, we just expose this helper,
// but if we add more attrs we can expose `FieldAttrs` properly:
```